pkgs/os-specific/linux/audit/default.nix

```diff
@@ -37,40 +37,9 @@ stdenv.mkDerivation rec {
   ];
 
   enableParallelBuilding = true;
-
-  # TODO: Remove the musl patches when
-  #         https://github.com/linux-audit/audit-userspace/pull/25
-  #       is available with the next release.
   patches = [
-    ./patches/weak-symbols.patch
-    (fetchpatch {
-      # upstream build fix against -fno-common compilers like >=gcc-10
-      url = "https://github.com/linux-audit/audit-userspace/commit/017e6c6ab95df55f34e339d2139def83e5dada1f.patch";
-      sha256 = "100xa1rzkv0mvhjbfgpfm72f7c4p68syflvgc3xm6pxgrqqmfq8h";
-    })
+    ./patches/fix-static.patch
```

Comment:
```suggestion
    ./fix-static.patch
```

Reply:
Fixed in: 50c40a8f6ab80600cf095855ecd7aa972cb6eac3